### DISCUSSION TITLE
clickable internal/network module type

### DISCUSSION THREAD

**Discussion by yumyumb612** [polybar/polybar#2982] [Q&A] [answered]
label-click and click-left does not work under that type of module, but does work with custom/text. lol, am i missing smthn? heheh lil halp. thanks! :D

**Comment by patrick96** [comment, marked as answer]
The network module just doesn't have that setting, see #273 

The same can be achieved with [action tags](https://github.com/polybar/polybar/wiki/Formatting#action-a) though.

**Reply by yumyumb612** [reply]
totally worked, thanks much bro :D
